src/Commands/FetchImages.php

```diff
@@ -0,0 +1,180 @@
+<?php
+
+namespace Redberry\Spear\Commands;
+
+use Illuminate\Console\Command;
+
+class FetchImages extends Command
+{
+	/**
+	 * The name and signature of the console command.
+	 *
+	 * @var string
+	 */
+	protected $signature = 'spear:pull {image?}';
+
+	/**
+	 * The console command description.
+	 *
+	 * @var string
+	 */
+	protected $description = 'Fetch docker images';
+
+	/**
+	 * Languages array for fetching images.
+	 */
+	private array $languages = [
+		'node',
+		'cpp',
+		'csharp',
+		'go',
+		'java',
+		'perl',
+		'php',
+		'python',
+		'ruby',
+		'rust',
+	];
+
+	/**
+	 * Execute the console command.
+	 */
+	public function handle()
+	{
+		if ($this->argument('image'))
+		{
+			$this->fetchImage($this->argument('image'));
+		}
+		else
+		{
+			$this->fetchAllImages();
+		}
+	}
+
+	/**
+	 * Checks if it exists images
+	 */
+	private function fetchImage($image)
+	{
+		switch ($image) {
+			case 'node':
+				$nodeVersions = [
+					'14' => 'node:14',
+					'16' => 'node:16',
+					'17' => 'node:17',
+					'18' => 'node:18',
+				];
```

Comment:
You already have those image versions in the Handlers class, why don't u use that instead of creating a new one. It's code duplication.

make Handler docker image versions public and static to be available to get those data outside of the class, and use those here.